package.json

```diff
@@ -49,7 +49,6 @@
   },
   "dependencies": {
     "chalk": "4.1.2",
-    "generator-jhipster": "7.8.1",
```

Comment:
I don't think you want to remove this. I'd add it back in and see if the CI tests pass. If they do, try upgrading it to 7.9.2 (the latest release).

Reply:
Okay got it. I'll try again.